style/js/shield_defs.js

```diff
@@ -1,619 +1,634 @@
+        fillColor,
+        strokeColor,
+        ref,
+        radius,
+        outlineWidth,
+        rectWidth
+      ),
+    textLayoutConstraint: (spaceBounds, textBounds) =>
+      ShieldText.roundedRectTextConstraint(spaceBounds, textBounds, radius),
+    padding: {
+      left: 3,
+      right: 3,
+      top: 3,
+      bottom: 3,
+    },
+    maxFontSize: 16,
+    textColor: textColor,
+  };
+}
+
+export function loadShields(shieldImages) {
+  // Multi-use shields
+
+  //Diamond shields
+  var diamondShield = {
+    backgroundImage: shieldImages.shield40_us_nc,
+    textColor: "black",
+    padding: {
+      left: 5,
+      right: 5,
+      top: 5,
+      bottom: 5,
+    },
+  };
+
+  var usMUTCDCountyShield = {
+    backgroundImage: shieldImages.shield40_us_county,
+    textColor: "#f7d117",
+    padding: {
+      left: 3,
+      right: 3,
+      top: 3,
+      bottom: 6,
+    },
+  };
+
+  shields["US:I"] = {
+    backgroundImage: [
+      shieldImages.shield40_us_interstate_2,
+      shieldImages.shield40_us_interstate_3,
+    ],
+    textLayoutConstraint: ShieldText.southHalfellipseTextConstraint,
+    textColor: "white",
+    padding: {
+      left: 3.5,
+      right: 3.5,
+      top: 5,
+      bottom: 4,
+    },
+  };
+
+  shields["US:I:Business:Loop"] = {
+    backgroundImage: [
+      shieldImages.shield40_us_interstate_business_2,
+      shieldImages.shield40_us_interstate_business_3,
+    ],
+    textLayoutConstraint: ShieldText.southHalfellipseTextConstraint,
+    textColor: "white",
+    padding: {
+      left: 3,
+      right: 3,
+      top: 5,
+      bottom: 5,
+    },
+  };
+
+  shields["US:I:Business:Spur"] = shields["US:I:Business:Loop"];
+
+  //US Highways
+  var shield_us_us = [
+    shieldImages.shield40_us_us_2,
+    shieldImages.shield40_us_us_3,
+  ];
+  var padding_us_us = {
+    left: 2,
+    right: 2,
+    top: 4,
+    bottom: 5,
+  };
+
+  shields["US:US"] = {
+    backgroundImage: shield_us_us,
+    textColor: "black",
+    padding: padding_us_us,
+  };
+
+  shields["US:US:Truck"] = {
+    backgroundImage: shield_us_us,
+    textColor: "black",
+    padding: padding_us_us,
+    modifiers: ["TRK"],
+  };
+
+  shields["US:US:Business"] = {
+    backgroundImage: shield_us_us,
+    textColor: "black",
+    padding: padding_us_us,
+    modifiers: ["BUS"],
+  };
+
+  shields["US:US:Alternate:Truck:Business"] = {
+    backgroundImage: shield_us_us,
+    textColor: "black",
+    padding: padding_us_us,
+    modifiers: ["ALT", "TRK", "BUS"],
+  };
+
+  shields["US:US:Historic"] = {
+    backgroundImage: shield_us_us,
+    textColor: "black",
+    padding: padding_us_us,
+    colorLighten: "#613214",
+  };
+
+  shields["US:AZ"] = {
+    backgroundImage: [
+      shieldImages.shield40_us_az_2,
+      shieldImages.shield40_us_az_3,
+    ],
+    textColor: "black",
+    padding: {
+      left: 4,
+      right: 3,
+      top: 3,
+      bottom: 4,
+    },
+  };
+  shields["US:AZ:Loop"] = shields["US:AZ"];

```

Comment:
This causes the Loop shields to look identical to the normal state route shields. Since the loop routes numerically overlap, it may be important for users to know that a loop route is a loop route. To display the word “Loop” on top of the state shape, set the object’s `modifiers` property to an array containing the string `Loop`, as with the `US:US:Business` definition above. To keep the `US:AZ:Loop` definition from affecting the `US:AZ` definition, you’ll need to either duplicate the whole object or call [`Object.assign()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign).

Reply:
I would suggest making the loop modifer `LOOP` in all-caps.